### ISSUE TITLE
can not find LayerNormalization

### ISSUE THREAD

**Issue by lindadamama** [SciSharp/TensorFlow.NET#875]
can not find LayerNormalization

**Comment by lindadamama** [comment]
missing layerNormalization
However, Torchsharp's APIS are complete
https://github.com/tensorflow/tensorflow/tree/v2.7.0/tensorflow/python/keras/layers/normalization
https://github.com/dotnet/TorchSharp/tree/main/src/TorchSharp/NN/Normalization

**Comment by Oceania2018** [comment]
Same as https://github.com/SciSharp/TensorFlow.NET/issues/782
